cli.js

```diff
@@ -14,7 +15,8 @@ const cli = meow(`
 	  211
 `);
 
-const input = cli.input[0];
+const input = cli.input[0] || cli.flags.pretty;
+const bytesFlag = cli.flags.pretty;
```

Comment:
I don't really see the point of this variable. It doesn't make things any shorter or clearer.